### ISSUE TITLE
Uncaught SyntaxError: Unexpected token export

### ISSUE THREAD

**Issue by zombor** [bootstrap-vue/bootstrap-vue#56]
Getting this error when trying to use individual components as described in the docs. From the console, it says the file is: `webpack:///./~/bootstrap-vue/utils/helpers.js?fc5f`

![bootstrap-vue](https://cloud.githubusercontent.com/assets/170538/21582088/20b18f3a-d010-11e6-9c9f-2d40860bbb6c.png)

I'm using Vue `2.1.8` and `latest` bootstrap-vue.

**Comment by pi0** [comment]
Hey @zombor , Would you please check your project's `.babelrc` file? It should contain needed presets as you are directly using this library inside your project. it should be something like this:
```js
  "babel": {
    "presets": [
      [
        "es2015"
      ]
    ]
  }
```

**Comment by zombor** [comment]
Yep, I have that:

```
{
  "presets": ["es2015", "stage-2"],
  "plugins": ["transform-runtime"],
  "comments": false
}
```

I didn't mention in the original post, but if I use `Option 1` (the whole package) it works fine. This only happens when I try and use it with individual components.

**Comment by pi0** [comment]
I think problem is that webpack babel-loader is ignoring to transform js files in `node_modules` so this happens. Until fix please using compiled version as no cost :)) (it is approx ~20k minified)

**Comment by acoard** [comment]
What do you mean by the compiled version, where can I download this?

Do you mean the `Vue.use(BootstrapVue)` command?  That throws the exact same error for me.

**Comment by pushamaku** [comment]
I had the same issue when I started using 0.4.1, it only worked if I used the whole package and not individual components. In 0.5.1 both methods do not work. Had to revert to 0.4.1 with bootstrap@4.0.0-alpha.5.

**Comment by pi0** [comment]
@pushamaku I'v just released 0.5.2 which uses compiled version in package.json so it should work out-of-the-bex  :)

**Comment by quantix-studio** [comment]
Same issue here with 0.6.0

**Comment by zombor** [comment]
Now this error happens on the "all components" install method since 0.6.1. I had to force my app to lock to 0.6.0 to get anything to work at all.

**Comment by pi0** [comment]
@zombor I'm so sorry for all inconveniences during our beta releases. As you mentioned if problem was appeared from 0.6.1 it was because of a breaking change (both nuxt.js for docs and webpack was overriding dist). I've just released `0.7.1` which should work well. please let me know if problem exists.
